<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AffichageArticles</class>
 <widget class="QWidget" name="AffichageArticles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>789</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>AffichageArticles</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbFiltre">
     <property name="title">
      <string>Filtrer par :</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblCategorie">
        <property name="text">
         <string>Catégorie :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbFiltreCategorie">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblGenre">
        <property name="text">
         <string>Genre :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbFiltreGenre">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblLibelle">
        <property name="text">
         <string>Libellé :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leFiltreLibelle"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnToutAfficher">
        <property name="text">
         <string>Tout Afficher</string>
        </property>
        <property name="icon">
         <iconset resource="../../../SeboApp.qrc">
          <normaloff>:/SeboApp/Resources/toutAfficher.png</normaloff>:/SeboApp/Resources/toutAfficher.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tvArticle">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../SeboApp.qrc"/>
 </resources>
 <connections/>
</ui>
